.github/ISSUE_TEMPLATE/template-graduation-application.md

```diff
@@ -0,0 +1,273 @@
+# $PROJECT Graduation Application
+
+This template provides the project with a framework to inform the TOC of their conformance to the Incubation Level Criteria. 
+
+
+Project Repo(s): $URL
+Project Site:  $URL
+Sub-Projects: $LIST
+Communication: $SLACK
+
+Project points of contacts: $NAME, $EMAIL
+
+## Graduation Criteria Summary for $PROJECT
+
+### Adoption Assertion
+
+_The project has been adopted by the following organizations in a testing and integration or production capacity:_
+* 
+
+### Criteria
+
+## Application Process Principles
+
+### Suggested
+
+N/A
+
+### Required
+
+- [ ] **Give a presentation and engage with the domain specific TAG(s) to increase awareness**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ]  **TAG provides insight/recommendation of the project in the context of the landscape**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ]  **All project metadata and resources are [vendor-neutral](https://contribute.cncf.io/maintainers/community/vendor-neutrality/).**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Review and acknowledgement of expectations for [graduated](sandbox.cncf.io) projects and requirements for moving forward through the CNCF Maturity levels.**		
```

Comment:
need to fix the link. The link does not work

Reply:
this is an error, there should be no link for this.